packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -112,7 +113,12 @@ class NgccLogger implements Logger {
 
   debug(..._args: string[]) { }
 
-  info(..._args: string[]) { }
+  info(..._args: string[]) {
+    const msg = _args.join(' ');
+    if (!msg.includes(this.alreadyProcessMsg)) {
+      process.stderr.write(`\n${msg}\n`);
```

Comment:
Maybe we should only print the `compiling` messages.

We should skip all others, such as skipping target already compiled and target entrypoint already compiled.

Reply:
@petebacondarwin should ngcc also log `Skipping` messages as debug?

Reply:
If you like 🤗

Reply:
I'm good with that approach then. @alan-agius4 WDYT?

Reply:
Please send a PR :-)

Reply:
Sounds good to me

Reply:
PR up at https://github.com/angular/angular/pull/30232, PTAL @petebacondarwin. I'll then change this PR to log all info messages instead.

Reply:
Approved